README.md

```diff
@@ -104,48 +111,54 @@ Codename:        bionic
 ```
 
 ## Stop an instance to save resources
+
 ```
 $ multipass stop dancing-chipmunk
 ```
 
-## Delete the instance
+## Delete an instance
+
 ```
 $ multipass delete dancing-chipmunk
 ```
 
-It will now show up as deleted:
+The instance will now show up as deleted:
+
 ```$ multipass list
+
 Name                    State             IPv4             Release
 snapcraft-asciinema     STOPPED           --               Ubuntu Snapcraft builder for Core 18
 dancing-chipmunk        DELETED           --               Not Available
 ```
 
-And when you want to completely get rid of it:
+If you want to completely get rid of it:
 
 ```
 $ multipass purge
 ```
 
 ## Get help
-```
 
+```
 multipass help
 multipass help <command>
 ```
 
-# Get involved!
+# Make your own build
+
+Here's a set of steps to make and run your own build of Multipass. 
```

Comment:
Maybe change to `Here's a set of steps to build the source code of Multipass. `

Reply:
Thanks, I'll use your suggestion :)